package.json

```diff
@@ -74,9 +78,13 @@
     "rimraf": "3.0.2",
     "rollup": "2.28.2",
     "rollup-plugin-auto-external": "2.0.0",
-    "sinon": "9.0.3"
+    "sinon": "9.0.3",
```

Comment:
Looks like there's a mix of exact versions and version ranges in dev deps and all version ranges in deps. Might be good to get on same page so I know your strategy to set them if I should add a new one down the road.

Reply:
renovate will enforce that dev deps are exact versions. i may not have set them as such in this branch, but they will get updated. renovate wont enforce either way (with my settings at least) for prod deps, but will honor whichever way it is set. my strategy is a little less than obvious in that context, so happy to chat in a more sync context about this whenever you want

Reply:
high-level answer is that dev deps are for maintainers only, so exact makes sense and forces more frequent updates. prod deps for packages have value in wider ranges to enable deduping on the consumer side. it is important to raise the lower bound when necessary (which a lot of packages dont do and i think cause a lot of the semver frustrations in the community). there are cases where i set prod deps to exact, which mostly come down to forcing updates, but those could be worth chatting more deeply about

Reply:
here is the [renovate PR](https://github.com/form8ion/eslint-config-extender/pull/31) to mark the dev-deps as exact, now that we merged to the `beta` branch, which [is watched based on my config](https://github.com/form8ion/renovate-config/blob/53f0682fcb795c3ebc7067dd9e0a07a839d1ba3a/js-package.json#L3)